src/cpp/session/modules/SessionCodeSearch.cpp

```diff
@@ -122,7 +122,7 @@ bool isWithinIgnoredDirectory(const FilePath& filePath, const std::vector<FilePa
          return true;
 
       // cmake build directory
-      if (parentPath.completeChildPath("cmake_install.cmake").exists())
```

Comment:
Since `cmake_install.cmake` is only written out for CMake build directories that actually have build commands. `CMakeCache.txt` is always written out.